services/src/test/resources/application.conf

```diff
@@ -0,0 +1,9 @@
+database {
+  gitlab {
+     url = "https://gitlab.com/api/v4/"
+     //Change AdminLogin for real GitLab login
```

Comment:
You should make this comments TODOs

Reply:
We will have one admin login for GitLab, and this login is a part of GitLab API to get to the repository path. And I mean in this comment some need to change "AdminLogin" for real Admin login to GitLab.

Reply:
What Ostap was saying, you can add `TODO` between `//` and `Change AdminLogin for real GitLab login`. It is a good idea and a common practice.
The benefit -- InteliJ could scan project for TODOs. And also static code analysers, which could be helpful for us in the future.

Reply:
Fixed!